Quick Paste/AppDelegate.swift

```diff
@@ -46,13 +47,34 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         event in
         if let strongSelf = self {
             print(strongSelf.globalBringUpMonitor)
-            strongSelf.viewController.keyDown(with: event!)
+//            strongSelf.viewController.keyDown(with: event!)
+            strongSelf.keyDown(with: event!)
         }
     }
+    func keyDown(with event: NSEvent) {
+        if event.keyCode == 12{
+            print("q detected")
+            switch event.modifierFlags.intersection(.deviceIndependentFlagsMask){
+            case[.control, .shift]:
+                print("control+shift");
+//                if(appDelegate == nil){
+//                    appDelegate = NSApplication.shared.delegate as! AppDelegate
+//                }
+                self.togglePopover(self.statusItem.button)
+            default:
+                break
+            }
+        }
+    }
+
```

Comment:
currently this is the only keyEvent need to be monitored globally. Separate it from other local event should quick fix the bug which copyOnReturn being called when menu is off.